workflow/state-machines.rst

```diff
@@ -32,7 +32,7 @@ Below is the configuration for the pull request state machine.
         framework:
             workflows:
                 pull_request:
-                    type: 'state_machine'
+                    type: 'single_state'
```

Comment:
In fact, the example was right before. `single_state` and `multiple_state` are the allowed values for the `type` property of the marking store.